deploy/nl/catalog.yaml

```diff
@@ -58,27 +58,22 @@ indexes:
     healthcheck_query: "Life expectancy"
   sdg_ft:
     store_type: MEMORY
-    embeddings_path: gs://datcom-nl-models/embeddings_sdg_2023_12_26_10_03_03.ft_final_v20230717230459.all-MiniLM-L6-v2.csv
+    embeddings_path: gs://datcom-nl-models/embeddings_sdg_2024_06_13_18_37_38.ft_final_v20230717230459.all-MiniLM-L6-v2.csv
     model: ft-final-v20230717230459-all-MiniLM-L6-v2
     healthcheck_query: "Hunger"
   undata_ft:
     store_type: MEMORY
-    embeddings_path: gs://datcom-nl-models/embeddings_undata_2024_03_20_11_01_12.ft_final_v20230717230459.all-MiniLM-L6-v2.csv
+    embeddings_path: gs://datcom-nl-models/embeddings_undata_2024_06_13_18_31_43.ft_final_v20230717230459.all-MiniLM-L6-v2.csv
     model: ft-final-v20230717230459-all-MiniLM-L6-v2
     healthcheck_query: "Hunger"
   undata_ilo_ft:
     store_type: MEMORY
-    embeddings_path: gs://datcom-nl-models/embeddings_undata_ilo_2024_05_15_11_18_05.ft_final_v20230717230459.all-MiniLM-L6-v2.csv
-    model: ft-final-v20230717230459-all-MiniLM-L6-v2
-    healthcheck_query: "Employment"
-  undata_dev_ft:
```

Comment:
@dwnoble I removed undata_dev_ft as it is the union of undata_ft and un_ilo_ft

Reply:
Nice! With this change, how do i call the /detect-and-fulfill endpoint to fetch both undata and undata_ilo? Currently the request looks like

```
POST /api/explore/detect-and-fulfill?q=employment%20in%20africa&client=ui_query
{
  "contextHistory":[],
  "dc":"undata_ilo",
  "disableExploreMore":""
}
``

Reply:
this should still work since you have "dc" param set!

Reply:
Got it - so the the undata_dev dc param still exists, it's now just defined by the combination of undata_ilo_ft and undata_ft 👍

Reply:
yes, so no need to build undata_dev index anymore